common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -104,12 +104,14 @@ eActionOp_CONTROL:
 
   ACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_REQUEST : 140
   ACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_RESPONSE : 141
-  ACTION_CONTROL_CHANNEL_SCAN_TRIGGERED_NOTIFICATION : 142
-  ACTION_CONTROL_CHANNEL_SCAN_RESULTS_NOTIFICATION : 143
-  ACTION_CONTROL_CHANNEL_SCAN_ABORT_NOTIFICATION : 144
-  ACTION_CONTROL_CHANNEL_SCAN_FINISHED_NOTIFICATION : 145
-
-  ACTION_CONTROL_ENUM_END: 146
+  ACTION_CONTROL_CHANNEL_DUMP_SCAN_RESULTS_REQUEST: 142
+  ACTION_CONTROL_CHANNEL_DUMP_SCAN_RESULTS_RESPONSE: 143
```

Comment:
I wanted to add them in relation to where they would be used in the scan flow
And since the `scan dump request` is an alternative version of the `scan trigger request` I thought it was best to add them after the scan trigger request (and response)